controller/news_controller.js

```diff
@@ -30,3 +31,27 @@ exports.patchArticleById = (req, res, next) => {
 }
 
 
+
+
```

Comment:
whitespace suggests the need to extract controllers (and models, for that matter) out into their own files: topics, users, articles etc.